service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/RemoveOrderControllerAboutToStartTest.java

```diff
@@ -108,14 +109,16 @@ void shouldAddAllNonSDRemovableOrdersToCaseDataInAllExpectedStates(State state)
                 buildListElement(generatedOrders.get(4).getId(), "order 5 - 12 September 2018"),
                 buildListElement(sealedCaseManagementOrders.get(0).getId(),
                     String.format("Sealed case management order issued on %s",
-                        formatLocalDateToString(dateNow(), "d MMMM yyyy"))),
```

Comment:
ahhh, it is better to leave it in this form for the test, if the constant changes for some reason then you would want to be able to  pick that up with failing tests